src/EventStore.Client/EventStore.Client.csproj

```diff
@@ -33,7 +34,7 @@
 	</ItemGroup>
 	<ItemGroup>
 		<Protobuf Include="../Protos/Grpc/*.proto" Exclude="../Protos/Grpc/operations.proto" Access="internal" GrpcServices="Client" ProtoRoot="../Protos/Grpc" />
-		<Compile Include="../EventStore.Grpc.Common/*.cs" />
+		<Compile Include="../EventStore.Client.Common/*.cs" />
 	</ItemGroup>
 	<ItemGroup>
 		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```

Comment:
We missed the InternalsVisible to Attribute here.
We can remove the `AssemblyInfo.cs` and just add the necessary attributes inline in the csproj.

```
	<ItemGroup>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.Tests</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.Operations</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.Projections.Tests</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.UserManagement.Tests</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.PersistentSubscriptions.Tests</_Parameter1>
		</AssemblyAttribute>
	</ItemGroup>
```